fastkafka/_application/app.py

```diff
@@ -716,6 +719,19 @@ def is_started(self) -> bool:
     def set_bootstrap_servers(self, bootstrap_servers: str) -> None:
         self._kafka_config["bootstrap_servers"] = bootstrap_servers
 
+    def set_kafka_broker(self, kafka_broker: str) -> None:
+        if kafka_broker not in self._kafka_brokers.brokers:
+            raise ValueError(
+                f"Given kafka_broker '{kafka_broker}' is not found in kafka_brokers, available options are {self._kafka_brokers.brokers.keys()}"
+            )
+
+        broker_to_use = self._kafka_brokers.brokers[kafka_broker]
+        bootstrap_servers = f"{broker_to_use.url}:{broker_to_use.port}"
+        logger.info(
+            f"set_kafka_broker() : Setting bootstrap_servers value to '{bootstrap_servers}'"
+        )
+        self._kafka_config["bootstrap_servers"] = bootstrap_servers
+
     async def __aenter__(self) -> "FastKafka":
```

Comment:
optional parameter kafka_broker_name: Optional[str] = None

if set, remember the old value in the instance variable. Set it back to the old value in __aexit__